stable/atlantis/README.md

```diff
@@ -46,6 +46,8 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `logLevel`                                  | Level to use for logging. Either debug, info, warn, or error.                                                                                                                                                                                                                                             | n/a     |
 | `orgWhiteList`                              | Whitelist of repositories from which Atlantis will accept webhooks. **This value must be set for Atlantis to function correctly.** Accepts wildcard characters (`*`). Multiple values may be comma-separated.                                                                                             | none    |
 | `repoConfig`                                | [Server Side Repo Configuration](https://www.runatlantis.io/docs/server-side-repo-config.html) as a raw YAML string. Configuration is stored in ConfigMap.                                                                                                                                                | n/a     |
+| `defaultTFVersion`                          | Default Terraform version to be used by atlantis server                                                                                                                                                                                                                                                   | `false` |
+| `allowForkPRs`                              | Allow atlantis to run on fork Pull Requests                                                                                                                                                                                                                                                               | n/a     |
```

Comment:
a slight nit, but the readme indicates the defaults are 
`defaultTFVersion=false; allowForkPRs=n/a`
but the implementation in the values file is opposite 
`defaultTFVersion=n/a; allowForkPRs=false`

Reply:
Oops, let me fix that

Reply:
Done